ACE/ace/INET_Addr.cpp

```diff
@@ -371,8 +371,11 @@ ACE_INET_Addr::set (u_short port_number,
 #endif /* ACE_HAS_IPV6 && ACE_USES_IPV4_IPV6_MIGRATION */
 
 #ifdef ACE_HAS_IPV6
+
+#ifndef AIX
```

Comment:
Could you give any suggestion on how to actually fix the problem instead of reverting all the changes and let the bug persists?